go/vt/vtgate/planbuilder/symtab.go

```diff
@@ -63,10 +63,7 @@ func newSymtab(vschema VSchema) *symtab {
 }
 
 // AddAlias adds a table alias to symtab.
-func (st *symtab) AddAlias(alias, astName sqlparser.TableIdent, table *vindexes.Table, rb *route) error {
-	if found := st.findTable(alias); found != nil {
```

Comment:
By removing this condition are we allowing duplicate symbols? we should override right?